zbus/src/connection/socket/command.rs

```diff
@@ -0,0 +1,126 @@
+use std::io;
+use std::os::fd::BorrowedFd;
+
+#[cfg(feature = "tokio")]
+use tokio::io::{AsyncReadExt, ReadBuf};
+
+use crate::process::{Child, ChildStdin, ChildStdout};
+
+use super::{ReadHalf, RecvmsgResult, Socket, Split, WriteHalf};
+
+/// A Command stream socket.
+///
+/// This socket may be used for communicating with a spawned child process via
+/// its standard input and output streams.
+#[derive(Debug)]
+pub(crate) struct CommandStream {
```

Comment:
`Stream` is quite a bit overloaded term in async Rust world and certainly also in zbus code. Let's just call it `Command` (the `socket` mod will ensure clarity, as long as it's not re-exported outside of it and it shouldn't IMO).